src/test/java/com/revature/servicestests/ContentServiceTest.java

```diff
@@ -18,25 +19,32 @@
 import org.springframework.test.annotation.Rollback;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
-import org.springframework.test.jdbc.JdbcTestUtils;
-import org.springframework.transaction.annotation.Transactional;
 
 import com.revature.entities.Content;
 import com.revature.entities.Link;
+import com.revature.repositories.ContentRepository;
```

Comment:
is this used?

Reply:
In that service, yes. It's used at a couple of points for forcible cleanup, but probably could be dropped thanks to the rollback annotation.